core/src/main/java/org/apache/brooklyn/core/catalog/internal/CatalogBomScanner.java

```diff
@@ -74,6 +80,17 @@ public void unbind(ServiceReference<ManagementContext> managementContext) throws
 
     public class CatalogPopulator extends BundleTracker<Iterable<? extends CatalogItem<?, ?>>> {
 
+        private final String BROOKLYN_PACKAGE_REGEX = "org.apache.brooklyn.*";
+
+        private final ConfigKey<List<String>> WHITELIST =
+            ConfigKeys.newConfigKey(new TypeToken<List<String>>(){}, "brooklyn.catalog.osgi.application.whitelist",
+                "CSV Whitelist of patterns to match bundle symbolic id if bundle is to be permitted to add " +
+                    "applications (templates) to the catalog", ImmutableList.of(BROOKLYN_PACKAGE_REGEX));
```

Comment:
If I am evaluating Brooklyn and load my own bundle in the container I'd expect that its `catalog.bom` is picked up out of the box. I think Brooklyn should accept all bundles, letting downstream distributions set their own policy.